pydm/widgets/base.py

```diff
@@ -296,6 +189,15 @@ def value_changed(self, new_val):
 
         self.update_format_string()
 
+    @pyqtProperty(int)
```

Comment:
This property needs to be hidden from Designer.

Reply:
Done.